recipes/tqdm/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "4.7.1" %}
+
+package:
+  name: tqdm
+  version: {{ version }}
+
+source:
+  fn: tqdm-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/t/tqdm/tqdm-{{ version }}.tar.gz
+  md5: 0d3bda7f215e406d9bfa66c65bb6eaf7
+
+build:
+  entry_points:
+    - tqdm=tqdm._main:main
+
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+
+test:
+  imports:
+    - tqdm
+
+  commands:
+    - tqdm --help
+
+  requires:
+    - coverage
+    - flake8
+    - nose
```

Comment:
If you're not going to run the tests, you probably don't need to include *any* of these in the requires section